backend/src/main/java/com/bakdata/conquery/io/jackson/Jackson.java

```diff
@@ -16,17 +14,20 @@
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
 
+import java.util.Locale;
+
 public class Jackson {
 	public static final ObjectMapper MAPPER;
 	public static final ObjectMapper BINARY_MAPPER;
-	
+
 	static {
 		MAPPER = configure(io.dropwizard.jackson.Jackson.newObjectMapper());
-		BINARY_MAPPER = configure(io.dropwizard.jackson.Jackson.newObjectMapper(new SmileFactory()));
+		BINARY_MAPPER = configure(io.dropwizard.jackson.Jackson.newObjectMapper(new SmileFactory()))
+								.disable(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, 										 SerializationFeature.WRITE_NULL_MAP_VALUES);
```

Comment:
Would it be too bad if we serialize empty collections for binaries?
If we can we can stand this bit of more data send/stored, i prefer the same configuration for both mappers.